converters/fromZigbee.js

```diff
@@ -730,6 +809,47 @@ const converters = {
             return payload;
         },
     },
+    lixee_haElectricalMeasurement: {
+        cluster: 'haElectricalMeasurement',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            const result = {};
+
+            const elements = [
+                /* 0x0305 */ 'totalReactivePower',
```

Comment:
I see a lot of values are using camel casing, snake casing is preferred instead.